AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift

```diff
@@ -141,6 +145,51 @@ class CompleteScreenViewModel: ObservableObject {
         exitRoute()
     }
     
+    private func refresh() {
+        guard ((session.stream.first?.sensorName.contains("AirBeam")) != nil) else { getMeasurementsAndDisplayData(); return }
+        var currentSensor = AirBeamStreamPrefix.airBeam3
+        if session.stream.first!.sensorName.contains("AirBeam2") { currentSensor = .airBeam2 }
```

Comment:
Is it possible, that there would be no stream for this session? Maybe we could add a guard before to check if session.stream.first is not nil?